pyproject.toml

```diff
@@ -103,7 +103,7 @@ sphinx = { version = "*", optional = true }
 sphinx_rtd_theme = { version = "*", optional = true }
 sphobjinv = { version = "*", optional = true }
 sphinx-gitref = { version = "*", optional = true }
-requests-cache = ">=0.9.2" # This version introduced requests_cache.cache_control.NEVER_EXPIRE
+requests-cache = ">=0.9.7" # This version uses requests_cache.policy.expiration
```

Comment:
It seems that as of `requests_cache>=1.0.0` that it is `requests_cache.policy.expiration`, but for `0.9.8` and before it is `requests_cache.expiration`... :) 

https://github.com/requests-cache/requests-cache/commit/7ebf9df7ae2534bad66dc4f102993f5fb6d789b2#diff-c806e1a8ff2e5be9541e2a7576876bbc549a6c0acb10a18125e33eb83f69324f

![image](https://user-images.githubusercontent.com/1482100/228587312-4438c547-b336-40f7-b7e4-c1e054bed874.png)
![image](https://user-images.githubusercontent.com/1482100/228587349-84cce096-0754-49ca-82d8-aa12b8a1d31a.png)

I noticed because our CI tests are using `nitpick==0.33.0` and `requests-cache==0.9.8` and failing with this error
```
ModuleNotFoundError: No module named 'requests_cache.policy.expiration'
```

Reply:
Thanks for the report!
I missed the version number, I thought I had tested with `0.9.8`.

I'll release a new version once this is merged: https://github.com/andreoliwa/nitpick/pull/564